src/main/java/frc/robot/robotcontainers/Robot2025Container.java

```diff
@@ -112,7 +112,7 @@ private void configureButtonBindings() {
     driver.back().onTrue(SwerveSubsystem.getInstance().setHeadingCommand(new Rotation2d(0)));
 
     //slow mode
-    driver.leftTrigger().onTrue(Commands.runOnce(() -> TeleopSwerve.setSpeeds(TeleopSpeeds.SLOW)))
+    driver.leftTrigger().whileTrue(Commands.runOnce(() -> TeleopSwerve.setSpeeds(TeleopSpeeds.SLOW)))
```

Comment:
This could be an improvement for slow mode.

Reply:
This was a bug that we identified and fixed at Earl of March last night, tested and the drivers say this improves slow mode.

Reply:
I don't think Jacob was there yesterday. If other students tested it, that is great.